package.json

```
@@ -110,7 +109,9 @@
     "eslint-plugin-github": "4.1.3",
     "eslint-plugin-jest": "24.3.6",
     "eslint-plugin-jsx-a11y": "6.4.1",
+    "eslint-plugin-mdx": "1.15.1",
     "eslint-plugin-prettier": "3.4.0",
+    "eslint-plugin-primer-react": "0.4.1",
```

I think this needs to be bumped to 0.4.2
```suggestion
    "eslint-plugin-primer-react": "0.4.2",
```

---

Also, thank you for moving this into devDependencies!